tests/test_flow/test_persistence.py

```diff
@@ -21,6 +21,7 @@ def read(self, path, extension):
 # It would be nice to move the builder setup into fixtures, but since we need
 # to access the bound functions as well (to check the number of times they were
 # called), it's easiest to just have one long test.
+@pytest.mark.no_parallel
```

Comment:
These tests fail now because the memoized entities can be computed multiple times. This note in deriver should really have been added with this change.

> NOTE 1: This makes non-persisted entities compute here as well
   as in executor pool. We need to keep track of what needs to be
   computed in main process vs subprocess when entity is not persisted.